config/scale/cluster/filesystem/kustomization.yaml

```diff
@@ -5,6 +5,7 @@ commonLabels:
   app.kubernetes.io/name: cluster
 
 resources:
+  - scale_v1beta1_filesystem_vdisknsd.yaml
```

Comment:
hummm.... looks like i missed this cleanup

Reply:
You got it.